ega-data-api-res/pom.xml

```diff
@@ -241,7 +261,8 @@
                         <configuration>
                             <imageName>${dockerRegistry}/res:${image.version}</imageName>
                             <baseImage>java:8</baseImage>
-                            <entryPoint>["java", "${debug.config}${debug.res.port}", "-jar", "/${project.build.finalName}.jar"]
+                            <entryPoint>["java", "${debug.config}${debug.res.port}", "-jar",
+                                "/${project.build.finalName}.jar"]

```

Comment:
Because of formatting, I think that the  "/${project.build.finalName}.jar" in newline will not work properly in debug mode, It should be on the same line. Can you check if it work with this?

Reply:
Oh, I missed that. You are right, it should be on the same line, I'll fix it.

Reply:
Done.